lesson04/assignment/index.html

```diff
@@ -9,7 +9,18 @@
 </head>
 <body>
   <main class="main">
-
+    <div class="box">
+      <img class="responsive-image" class="responsive-image-1" src="./images/yema.jpeg" alt="image of a lawyer">
+      <!--<img class="responsive-image" class="responive-image-2" src="/images/top-lawyers.jpeg" alt="image of a lawyer"> -->
+    </div>
+     <!-- filler image, make sure to find a better picture in black and white-->
+     <div class="header-text" class="col-md-12 text-center">
+      <h1 class="animate-charcter"> Law Offices of Yema Measho </h1>
+  
+      <h3> Home </h3>
+      <h3> About Us</h3> 
+      <h2> <span></span></h2>
```

Comment:
Might be worth adding a comment here, or at least a class name to the h2, so other developers that may be working on the project can easily understand what is happening.  Using something like <span class="animatingText"> and then in your CSS target it with span.animatingText{ } would also prevent this animation from occurring on all span tags in your project.